app/controllers/events_controller.rb

```diff
@@ -0,0 +1,32 @@
+class EventsController < ApplicationController
+  before_action :current_user, only: %i[index new show create]
+
+  def index
+    @events = Event.all
```

Comment:
- [ ] Define two variables in the index action, `past_events` and `future_events` to be used at the index view here.